src/MorseL.Client/Connection.cs

```diff
@@ -570,6 +570,33 @@ await receiveDelegator(new ConnectionContext(_clientWebSocket, stream))
                             .ConfigureAwait(false);
                     }
                 }
+                catch (WebSocketException e) when (e.Message == "The remote party closed the WebSocket connection without completing the close handshake.")
+                {
+                    _logger.LogWarning("Observed WebSocketClosedException - likely OK");
```

Comment:
You may want this warning to say "WebSocketException" instead of "WebSocketClosedException".

Reply:
Fixed.